src/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ if (LEAN_SPECIAL_VERSION_DESC)
 elseif (NOT LEAN_VERSION_IS_RELEASE)
   string(APPEND LEAN_VERSION_STRING "-pre")
 endif()
+set(LEAN_MANUAL_ROOT "") # Set this to the correct location for releases and release candidates. Include the trailing slash.
```

Comment:
Similarly here, I think the comment should tell the human what the URL should be, if they need to set it by hand!

Reply:
It's now computed. We can always set one by hand if we have to, but no need to do it every time.